runtimes/common/src/benchmarks.rs

```diff
@@ -15,10 +15,17 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use core::marker::PhantomData;
+use pallet_treasury::ArgumentsFactory;
 use parity_scale_codec::{Decode, Encode};
 use scale_info::TypeInfo;
 use sp_std::vec::Vec;
 
+use crate::{constants::KILT, AccountId};
```

Comment:
I think we should stick to `AccountId` and remove the definition of `AccountIdOf` below.

Reply:
I refactored the benchmarks in: [f7e5129](https://github.com/KILTprotocol/kilt-node/pull/655/commits/f7e51296002107e04d22f4317f75d4b3a6822194) 

I couldn't stick with AccountId. Rust was complaining otherwise.

Reply:
Will resolve this, I'm opening a second review anyway.